docs/guides/lab-assembly/pcengines.md

```diff
@@ -0,0 +1,189 @@
+
+* serial connection to the platform,
+* controlling power supply,
+* enabling basic power actions with the platform (power off/power on/reset),
+* external flashing with the RTE,
+* CMOS reset.
+
+### Serial connection
+
+The method of setting and using serial connection is described in the
+[Serial connection guide](../../transparent-validation/rte/v1.1.0/serial-port-connection-guide.md).
+
+### Power supply controlling
+
+Power supply controlling is performed with the relay module on RTE
+connected to one of RTE GPIOs. Power operation should be performed using
+the `rte_ctrl` script implemented in `meta-rte` (OS image dedicated to the
+RTE platform).
+
+To toggle the power supply use the below command:
+
+```bash
+rte_ctrl rel
+```
+
+Or using [OSFV cli](https://github.com/Dasharo/osfv-scripts) from remote
+machine:
+
+```sh
+osfv_cli rte --model=<APU2/3/4/6> --rte_ip <ip_address> rel tgl
+```
+
+### Basic power operations
+
+Basic power operations should be performed based on the `rte_ctrl` script
+implemented in `meta-rte` (OS image dedicated to the RTE platform). To perform
+basic power operations use the below-described commands:
+
+1. Turn on the platform:
+
+    ```bash
+    rte_ctrl pon
+    ```
+
+1. Turn off the platform:
+
+    ```bash
+    rte_ctrl poff
+    ```
+
+1. Reset the platform:
+
+    ```bash
+    rte_ctrl reset
+    ```
+
+Or using OSFV cli from remote machine:
+
+```sh
+osfv_cli rte --model=<APU2/3/4/6> --rte_ip <ip_address> pwr on
+```
+
+```sh
+osfv_cli rte --model=<APU2/3/4/6> --rte_ip <ip_address> pwr off
+```
+
+```sh
+osfv_cli rte --model=<APU2/3/4/6> --rte_ip <ip_address> pwr reset
+```
+
+### CMOS clear
+
+To clear the CMOS, turn off the power with Sonoff or relay and use the
+following commands:
+
+```bash
+echo 1 > /sys/class/gpio/gpio412/value
+sleep 10
+echo 0 > /sys/class/gpio/gpio412/value
+```
+
+### S1 button
+
+S1 button is used to re-enable disabled serial console. More information about
+this feature can be found in:
+
+* [S1 switch button properties](https://pcengines.github.io/apu2-documentation/gpios/#s1-switch-button)
+* [Disable serial console and enable with S1 button](https://pcengines.github.io/apu2-documentation/theory-of-operation/#pc-engines-apu-firmware-features)
+
+To expose this feature to OSFV we have add additional wire between:
+
+| RTE J11 pin       | PC Engines J5 pin   |
+|:-----------------:|:-------------------:|
+| 7                 | 5 (MODESW#)         |
+
+### USB devices
+
+Since some issues with USB controllers may only happen on select USB ports,
+it's important to plug in USB devices to all 2 USB ports of the board.
+
+### SD card
+
+Insert at least 16GB SD card into platform SD card socket.
+
+### BIOS WP
+
+BIOS WP (aka SPIWP#) pin can be used to enable or disable write protection.
+Feature was described in [sortbootorder
+documentation](https://github.com/pcengines/sortbootorder?tab=readme-ov-file#bios-wp-option).
+
+| RTE J11 pin       | PC Engines J5 pin   |
```

Comment:
SPIWP is on J2 on apu2. The pin header marking may be different across variants, so we have to check all schematics